hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -336,6 +337,22 @@ public boolean doesTableExist(String tableName) {
     }
   }
 
+  /**
+   * @param databaseName
+   * @return  true if the configured database exists
+   */
+  public boolean doesDataBaseExist(String databaseName) {
```

Comment:
It's an open topic. IMHO, `checkDatabaseExist` is a better method name. WDYT?

Reply:
I refer to other names in the source code that also have similar functions
![749cea6a2c22aeb14200eb89bfd4bde](https://user-images.githubusercontent.com/25769285/94353758-73f66c80-00a7-11eb-96a6-1c2d7cbf8388.png)

Reply:
What do you think？ if checkDatabaseExist is better, I am very willing to modify
@yanghua

Reply:
> What do you think？ if checkDatabaseExist is better, I am very willing to modify
> @yanghua

OK, keep the current name.